vuepress/docs/next/tutorials/devops/backing-restoring-keycloak.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebarDepth: 2
+---
+
+# Backing up and Restoring Keycloak
+
+
+This tutorial describes how to backup and restore an active Entando Identity Management Keycloak instance when you need to migrate or restore the realm to another environment.
+
+## Prerequisites:
+A Keycloak instance running on Entando with a database management system (DBMS) in the backend. This procedure will not work with an embedded database. 
+
+## Create a Backup of Keycloak Realm
+1. From the command line, get the pod name of the Keycloak server you need to backup.
```

Comment:
"wish to backup" to deemphasize urgency and highlight the versatility of this action ??

Reply:
Maybe also note this - "In a quickstart environment the pod name will begin with quickstart-kc-server-deployment-"